rbd/rbd.go

```diff
@@ -463,8 +463,15 @@ func (image *Image) Copy(args ...interface{}) error {
 		return RbdErrorImageNotOpen
 	}
 
+	if !(len(args) == 1 || len(args) == 2) {
```

Comment:
Oy. If it was this broken I wonder if we shouldn't just abandon this approach all together for something more idomatic. Generally, I have not been pushing for bigger refactorings of the api because I assume the apis work and are being used (and thus call for a gentler approach) but this is bad enough that I feel we can take this as an opportunity. How about breaking this patch out of this PR into its own so we can discuss this seperately because most of the other patches in this series are looking good so far.

Reply:
Okay, I'll split the `Image.Copy()` part out of this and rewrite the function completely.